app/validators/wpcc/salary_threshold_validator.rb

```diff
@@ -0,0 +1,22 @@
+module Wpcc
+  class SalaryThresholdValidator < ActiveModel::Validator
+    SALARY_THRESHOLDS  = [5876.00, 489.67, 452.00, 113.00].freeze
+    SALARY_FREQUENCIES = %w[year month fourweeks week].freeze
+    LOW_SALARY_THRESHOLDS = SALARY_FREQUENCIES.zip(SALARY_THRESHOLDS).to_h
```

Comment:
A solution to this:

On the validator:

```ruby
class SalaryThresholdValidator 
  LOW_SALARY_THRESHOLDS = 
   YAML.load_file(Wpcc::Engine.root.join('config','low_salary_thresholds.yml'))
end
```

On the form:
```ruby
class YourDetailsForm
  SALARY_FREQUENCIES = 
   YAML.load_file(Wpcc::Engine.root.join('config','low_salary_thresholds.yml')).keys
end
```

@willhall88 & @Tr4pSt3R wdyt?